File: mRemoteV1/CS/Forms/OptionsPages/OptionsPage.cs
// VBConversions Note: VB project level imports
using System.Collections.Generic;
using System;
using AxWFICALib;
using System.Drawing;
using System.Diagnostics;
using System.Data;
using AxMSTSCLib;
using Microsoft.VisualBasic;
using System.Collections;
using System.Windows.Forms;
// End of VB project level imports

using System.ComponentModel;


namespace mRemoteNG.Forms.OptionsPages
{
	public class OptionsPage : UserControl
		{
			public OptionsPage()
			{
				InitializeComponent();
			}
			
#region Public Properties
			[Browsable(false)]public virtual string PageName {get; set;}
			
			public virtual Icon PageIcon {get; set;}
#endregion
			
#region Public Methods
			public virtual void ApplyLanguage()
			{
				
			}
			
			public virtual void LoadSettings()
			{
				
			}
			
			public virtual void SaveSettings()
			{
				
			}
			
			public virtual void RevertSettings()
			{
				
			}
#endregion
		}
}

File: mRemoteV1/Forms/OptionsPages/OptionsPage.vb
Imports System.ComponentModel

Namespace Forms.OptionsPages
    Public Class OptionsPage
        Inherits UserControl

#Region "Public Properties"

        <Browsable(False)>
        Public Overridable Property PageName As String

        Public Overridable Property PageIcon As Icon

#End Region

#Region "Public Methods"

        Public Overridable Sub ApplyLanguage()
        End Sub

        Public Overridable Sub LoadSettings()
        End Sub

        Public Overridable Sub SaveSettings()
        End Sub

        Public Overridable Sub RevertSettings()
        End Sub

#End Region
    End Class
End Namespace